content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc

```diff
@@ -0,0 +1,60 @@
+
+
+== Install the YubiHSM 2 Tools and Software
+
+To complete the procedures in this guide, install the YubiHSM 2 tools and software.
+
+[TIP]
+=====
+A generic prompt, *$*, is used in command line examples in this document. Depending on your command line application, your prompt may be different.
+=====
+
+*Step 1* Unzip the downloaded https://developers.yubico.com/YubiHSM2/Releases/[archives of the SDK] containing the YubiHSM libraries and tools and move the contents to an appropriate location.
+
+*Step 2* On your Windows system, run both installers:
+
+         * yubihsm-cngprovider-windows-amd64.msi (YubiHSM Key Storage Provider)
+         * yubihsm-connector-windows-amd64.msi (YubiHSM Connector for Windows)
+
+
+=== Default YubiHSM 2 Default Device Configuration
+
+The YubiHSM 2 device comes with a default, factory-installed authentication key that has the password `password`. As part of the configuration in this guide, the default authentication key will be destroyed. If the device is reset to its default configuration, any objects stored on the device that are not factory-installed are destroyed. Reset instructions can be found in https://developers.yubico.com/YubiHSM2/Usage_Guides/Factory_reset.html[Factory Reset].
```

Comment:
Please make this a relative link of "link:../Factory_reset.adoc"

Reply:
Done